### PR TITLE
Add functions related to photometry

### LINKED ISSUE #16: Add photometry related scripts
Move from SIMPLE the scripts which:
- Add photometry 
- Query SVO
- assign UCD

### PR DESCRIPTION
**Pull request by kelle**
closes #16 .

- Moves functions and tests from SIMPLE to this package
  - `ingest_photometry`
  - `ingest_photometry_filter`
  - `fetch_svo`
  - `assign_ucd`

- Implements use of `conftest.py` to setup the test database for use by all the tests and a `db` fixture.

**TODO:**
- [x] test ingest_photometry
- [x] test ingest_photometry_filter.

### CODE REVIEW

**Comment by kelle** [comment]
On hold until Photometry table is added to the template database. https://github.com/astrodbtoolkit/astrodb-template-db/issues/27

File: .gitignore [hunk 1]
```diff
@@ -160,4 +165,5 @@ cython_debug/
 #.idea/
 
 # Additional
-*.sqlite
\ No newline at end of file
+.vscode/
+.DS_Store
```
Comment (arjunsavel) [inline .gitignore:169]: does this catch all `DS_Store`s in all subfolders, too?

File: astrodb_scripts/photometry.py [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+    Returns
+    -------
+    ucd: str
+        UCD string
+
+    """
+    wave_eff_quantity.to(u.Angstrom)
+    wave_eff = wave_eff_quantity.value
+
+    if 3000 < wave_eff <= 4000:
```
Comment (arjunsavel) [inline astrodb_scripts/photometry.py:378]: it'd probably be a bit more readable to have this in a dictionary

**Review by arjunsavel — COMMENTED** [review]
Looks great! Just a few small things.

File: astrodb_scripts/photometry.py (hunk 2 above)
Comment (kelle) [reply astrodb_scripts/photometry.py:378]: template: https://stackoverflow.com/questions/53853276/mapping-range-of-values-to-string

File: .gitignore (hunk 1 above)
Comment (kelle) [reply .gitignore:169]: I believe so!

**Review by arjunsavel — APPROVED** [review]

**Pull request merged**